pom.xml

```diff
@@ -53,11 +37,20 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.epam.tat.module4</groupId>
-            <artifactId>calculator</artifactId>
-            <version>1.0</version>
-            <scope>system</scope>
-            <systemPath>${project.basedir}/src/main/resources/calculator-1.0.jar</systemPath>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>${selenium.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.github.bonigarcia</groupId>
+            <artifactId>webdrivermanager</artifactId>
+            <version>${bonigarcia.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-nop</artifactId>
+            <version>1.7.13</version>
```

Comment:
Why don't you use a variable for the version of slf4j? And where do you use slf4j?

Reply:
If there is no slf4j dependency in the pom.xml, it gives me a  SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder" warning, so I figured I should add it.